hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientConnection.java

```diff
@@ -33,7 +33,7 @@
 import com.hazelcast.nio.Address;
 import com.hazelcast.nio.Connection;
 import com.hazelcast.nio.ConnectionType;
-import com.hazelcast.nio.OutboundFrame;
+import com.hazelcast.internal.networking.OutboundFrame;
```

Comment:
The import is out-of-order, could you please organize them? Same for a bunch of other classes.